.gitignore

```diff
@@ -155,3 +155,7 @@ renv
 # Node Version Management
 web/.nvmrc
 node_modules
+
+# PDF generation
+api/app/utils/daily_rendered.html
```

Comment:
Kinda redundant? But I'm guilty of putting junk in here too.

Reply:
How do you mean redundant?

Reply:
Oh - I mean you won't be generating these files inside the project once you're using the temporary file/folder approach.

Reply:
all in memory now as of https://github.com/bcgov/wps/pull/1763/commits/4e333fd132ab77cf21de524d5282b9bd8f573941